frontend/src/components/admin/manageContent/formModals/ProjectFormModal.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { Form, Input, InputNumber, Select, Switch, Tooltip, Typography } from "antd";
+import { ApolloCache, useQuery } from "@apollo/client";
+import { QuestionCircleOutlined } from "@ant-design/icons/lib";
+import { FORM_RULES } from "../../../../util/util";
+import { CREATE_PROJECT_MUTATION, PROJECT_LIST_QUERY, UPDATE_PROJECT_MUTATION } from "../../../../types/Project";
+import ManageContentModal from "../ManageContentModal";
+import { FormModalProps } from "./FormModalProps";
+import { ALL_USERS_QUERY } from "../../../../types/User";
+
+const { Text } = Typography;
+
+const ProjectFormModal: React.FC<FormModalProps> = (props) => {
+  const { loading, data, error } = useQuery(ALL_USERS_QUERY);
+
+  if (error) {
+    return (
+      <>
+        <Text type="danger">Error: Unable to load user list.</Text>
+        <Text>{error?.message}</Text>
+      </>
+    );
+  }
+
+  const leadOptions = loading ? [] : data.users.map((user: any) => ({
+    label: `${user.preferredName} ${user.lastName}`,
+    value: user.id
+  }));
+
+  return (
+    <ManageContentModal
```

Comment:
When you create a project, click save, and then go to create another one, the old values aren't cleared in the modal.

Reply:
Fixed